build

```diff
@@ -0,0 +1 @@
+/net/soft_scratch/users/lzheng/openimageio/metadata/build
```

Comment:
This looks suspiciously like a mistakenly added file. Maybe it's responsible for all the tests suddenly failing?

Reply:
Yes, checking right now but definitely on me for being hasty!

Reply:
Hi Larry,

That should be removed now and just pushed with the latest updates, hopefully these will show all green soon!